test/app/views/core/SubscribeModal.spec.coffee

```diff
@@ -204,17 +204,17 @@ describe 'SubscribeModal', ->
         it 'calls hide()', wrapJasmine ->
           spyOn(@modal, 'hide')
           spyOn(me, 'fetch').and.returnValue(Promise.resolve())
-          yield @payPalButton?.click()
+          yield @modal.onClickStripeLifetimeButton()
           expect(@modal.hide).toHaveBeenCalled()
           expect(@getTrackerEventNames()).toDeepEqual(
             [ "Start Lifetime Purchase", "Finish Lifetime Purchase" ])
 
-      describe 'when the PayPal purchase response is 422', ->
+      describe 'when the Stripe purchase response is 402', ->
         beforeEach ->
-          @purchaseRequest.andReturn({status: 422, responseText: '{"i18n": "subscribe.paypal_payment_error"}'})
+          @purchaseRequest.andReturn({status: 402, responseText: '{}'})
 
-        it 'shows state "error"', wrapJasmine ->
-          yield @payPalButton?.click()
-          expect(@modal.state).toBe('error')
+        it 'shows state "declined"', wrapJasmine ->
+          yield @modal.onClickStripeLifetimeButton()
+          expect(@modal.state).toBe('declined')
           expect(@getTrackerEventNames()).toDeepEqual(
             [ "Start Lifetime Purchase", "Fail Lifetime Purchase" ])
```

Comment:
Why keep all but the `'uses Stripe'` test in the regional pricing describe block? The rest seems to just repeat the previous describe block.

Reply:
Good point, will remove.